extension/shepherd/popup.js

```diff
@@ -0,0 +1,55 @@
+/*
+ * Media variables to be fetched
+ */
+var mediaId = 0;
+const apiUrl = "http://localhost:4200/api";
+const media = new Request(
+  apiUrl + "/medias/" + mediaId,
```

Comment:
A method for building URLs would be nice (i.e.: `let mediaUrl = getApiRouteFor(mediaId)`)